loomi/regulariza-diagonal-client
# Feature/password modal

## Feature
 - Change PasswordModal
 - Forms for others modals
---

**What I did:**

- Change Password Modal
- Create separate form components for the rest of the modals
- implemented formik and validations for the forms of the modals

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/131438437-da1882d6-8956-4963-94ff-b4b2062fb1b1.png)
